tests/test_e2e_31_of_lldp_loop_detection.py

```diff
@@ -0,0 +1,129 @@
+import requests
```

Comment:
@gretelliz could you rename this file to `test_e2e_32_of_lldp.py`? I ended using 31 there PR #137, I should've probably used 32 there but let's use 32 here to simplify changes to get merged.

Reply:
Ok

Reply:
Thanks